### DISCUSSION TITLE
Ошибка VkNet.Exception.VkApiException: Form element not found

### DISCUSSION THREAD

**Discussion by DarkLiker** [vknet/vk#1559] [Q&A]
Здравствуйте!
Подскажите пожалуйста, делаю приложение в Visual studio 2022 Enterprize, на Xamarin Forms, взял вашу библиотеку VKNET за основу, при попытке сделать авторизацию через :
`vk.Authorize(new VkNet.Model.ApiAuthParams
                {
                    ApplicationId = 51711911,
                    Login = "***********",
                    Password = "*************",
                    Settings = Settings.All
                });`
                
                Вылетает ошибка: VkNet.Exception.VkApiException: Form element not found.
                
Так же ради интереса так как нигде не нашёл фикса или как самому пофиксить, попробовал этот же код для WIN FORM Майкрософт, на win form всё заработало отлично с 1 раза.
Искал информацию по Xamarin.Forms к сожалению так же не нашёл, подскажите пожалуйста как с этим бороться?
Так же прикрепляю фото ошибки с мобилки.
![photo_2023-07-24_21-23-25](https://github.com/vknet/vk/assets/34660698/28542d87-7d55-4e26-8a90-fac7f50d8bf5)
![photo_2023-07-24_21-23-28](https://github.com/vknet/vk/assets/34660698/fcad21ba-d458-4a57-a833-6ac1905d64ec)
![photo_2023-07-24_21-23-19](https://github.com/vknet/vk/assets/34660698/f3f020a0-b16c-4c4a-af9c-4cd0c42d04b4)
![photo_2023-07-24_21-23-23](https://github.com/vknet/vk/assets/34660698/9659344d-b111-4856-a99a-87e1bc36d5fe)

**Comment by OneCellDM** [comment]
Да  возникают  проблемы с авторизацией часто, т.к вк постоянно меняет многие вещи. 

Решение такое:
Сделайте авторизацию через браузер самостоятельно, открыв браузер и проведя пользователя по ссылке.

Пример ссылки для авторизации:

```
private const string AuthUrl =
           @"https://oauth.vk.com/oauth/authorize?client_id=APP_ID" +
           "&scope=1073737727" +
           "&redirect_uri=https://oauth.vk.com/blank.html" +
           "&display=mobile" +
           "&response_type=token" +
           "&revoke=1"; 
```
 Проведите авторизацию через браузер, а в конце, спарсите данные в URL адресе страницы.
 

 Создайте экземпляр и дайте ему Token и UserID:
```
 
 var api = new VkApi();
 api.Authorize(new ApiAuthParams { AccessToken = token, UserId = id });
```

Пользуюсь этим методом часто, всегда работает. без проблем. 

Если нужна помощь в реализации авторизации самостоятельно через браузер, можете написать мне, ссылки для связи указаны в профиле.
